grpc-bridge-example/README.md

```diff
@@ -198,15 +198,25 @@ that will be created in grpcToRest.example.grpc. To run it, rename it, move it t
 
 <pre><code>
       &lt;dependency&gt;
-		  &lt;groupId&gt;junit&lt;/groupId&gt;
-		  &lt;artifactId&gt;junit&lt;/artifactId&gt;
-		  &lt;version&gt;4.13.2&lt;/version&gt;
-	  &lt;/dependency&gt;
-	  &lt;dependency&gt;
-		  &lt;groupId&gt;org.jboss.resteasy&lt;/groupId&gt;
-		  &lt;artifactId&gt;resteasy-client&lt;/artifactId&gt;
-		  &lt;version&gt;6.2.3.Final&lt;/version&gt;
-	  &lt;/dependency&gt;
+          &lt;groupId&gt;io.grpc&lt;/groupId&gt;
+          &lt;artifactId&gt;grpc-netty-shaded&lt;/artifactId&gt;
+          &lt;version&gt;1.54.1&lt;/version&gt;
+      &lt;/dependency&gt;
+      &lt;dependency&gt;
+          &lt;groupId&gt;junit&lt;/groupId&gt;
+          &lt;artifactId&gt;junit&lt;/artifactId&gt;
+          &lt;version&gt;4.13.2&lt;/version&gt;
+      &lt;/dependency&gt;
+      &lt;dependency&gt;
+          &lt;groupId&gt;org.jboss.resteasy&lt;/groupId&gt;
+          &lt;artifactId&gt;resteasy-client&lt;/artifactId&gt;
+          &lt;version&gt;6.2.3.Final&lt;/version&gt;
+      &lt;/dependency&gt;
+      &lt;dependency&gt;
+          &lt;groupId&gt;org.wildfly.core&lt;/groupId&gt;
+          &lt;artifactId&gt;wildfly-controller-client&lt;/artifactId&gt;
+          &lt;version&gt;22.0.0.Beta1-SNAPSHOT&lt;/version&gt;

```

Comment:
I changed the version to: `19.0.0.Beta18` because the other version is not usable in my env.

Reply:
OK